api/app/signals/apps/api/views/source.py

```diff
@@ -4,18 +4,23 @@
 from django_filters.rest_framework import DjangoFilterBackend
 
 from signals.apps.api.serializers.source import SourceSerializer
-from signals.apps.signals.models import Source
+from signals.apps.signals.models import Signal, Source
 from signals.auth.backend import JWTAuthBackend
 
 
 class PrivateSourcesViewSet(DatapuntViewSet):
     """
-    Private ViewSet to display all sources in the database
+    Private ViewSet to display all sources, excluded the "onine" (Signal.SOURCE_DEFAULT_ANONYMOUS_USER) source
```

Comment:
spelling nits: "excluded" -> "excluding" and "onine" -> "online"

Reply:
Done